diff --git a/laf/CMakeLists.txt b/laf/CMakeLists.txt
index 697db42..e094e9d 100644
--- a/laf/CMakeLists.txt
+++ b/laf/CMakeLists.txt
@@ -42,13 +42,13 @@ endif()
 # Find libraries
 if(LAF_BACKEND STREQUAL "skia")
   include(FindSkia)
-else()
-  if(NOT FREETYPE_LIBRARIES)
-    find_package(Freetype)
-  endif()
-  if(NOT HARFBUZZ_LIBRARIES)
-    find_package(HarfBuzz)
-  endif()
+endif()
+
+if(NOT FREETYPE_LIBRARIES)
+  find_package(Freetype)
+endif()
+if(NOT HARFBUZZ_LIBRARIES)
+  find_package(HarfBuzz)
 endif()
 
 add_subdirectory(third_party)
